util/CMakeLists.txt

```diff
@@ -356,6 +356,25 @@ if(APPLE)
     set(output_dir "${CMAKE_CURRENT_BINARY_DIR}/util/mach")
     file(MAKE_DIRECTORY "${output_dir}")
 
+    get_property(archs TARGET crashpad_util PROPERTY OSX_ARCHITECTURES)
+    if(NOT archs)
+        if(IOS)
+            set(archs "arm64")
+        else()
+            set(archs "x86_64")
```

Comment:
Testing this locally, I had build errors whenever I tried to use x64 on ios, and arm64 on mac. I thought things are supposed to work. Maybe its a matter of specifying a different sysroot?